src/configuration/cloudformation.coffee

```diff
@@ -0,0 +1,89 @@
+#===============================================================================
+# CFo Template Configuration
+# Pull in the main API definition and assorted mixins to generate a
+# CloudFormation template.  Each mixin's template is merged into a large CFo
+# template that is attached to the main configuration object.
+#===============================================================================
+{resolve, basename} = require "path"
+{async, read, merge, readdir, isFile, last, exists} = require "fairmont"
+{yaml} = require "panda-serialize"
+_render = require "panda-template"
+preprocessors = require "./preprocessors"
+
+AWSTemplateFormatVersion = "2010-09-09"
+
+skyMixinsPath = resolve __dirname, "..", "..", "mixins"
+
+
+# Adapter to keep existing code working
+transitional = async (config, envName) ->
+  globals = merge config, {env: envName}
+  appRoot = process.cwd()
+  cfoTemplate = yield renderTemplate appRoot, globals
+  config.aws.cfoTemplate = JSON.stringify cfoTemplate
+  config
+
+# Returns an object, not a serialized string.
+renderTemplate = async (appRoot, globals) ->
+  Description = globals.description || "#{globals.name} - deployed by Panda Sky"
+  Resources = yield renderResources appRoot, globals
+
+  return {
+    AWSTemplateFormatVersion
+    Description
+    Resources
+  }
+
+# Finds and renders all mixins as the Resources for a CloudFormation Template.
+renderResources = async (appRoot, globals) ->
```

Comment:
Future reference: I've always wanted to distinguish between mixins and the “main” template, rather than having everything be a mixin. If we aren't going to do that, we should use different terminology, since there has to be a thing to mix into. Sky is prescriptive about APIs and Lambdas. Mixins augment that. Air Strike is prescriptive about using Lambdas and SQS (perhaps). Mixins would augment that. That said, maybe that's the wrong model. Maybe we want to be able to mix in Panda Sky?